File: uniquePath.js
/*The problem is to count all the possible paths from top left to bottom right
we have matrxix of row X Column*/
function uniquePath(row,column)
{
  if(row==1 || column==1) //returning condition
    return 1;
  else
    //recursive call to function
    return uniquePath(row-1,column)+uniquePath(row,column-1) ;
}
let r=prompt("enter rows");
let col=prompt("enter column");
console.log("Number of unique path:  "+uniquePath(r,col));

File: uniquepath.go
package main
import "fmt"
func unique(row int,col int) int{
	if row==1||col==1{
		return 1
	} else{
    fmt.Println(row)
    fmt.Println(col)
		return unique(row-1,col)+unique(row,col-1)
	}
}
func main(){
	var row,col int
	fmt.Printf("Enter row & column: ")
	fmt.Scanf("%d %d",&row,&col)
	res:=unique(row,col)
	fmt.Println("No. of path: ",res)
}
